src/main/scala/midas/models/dram/Util.scala

```diff
@@ -527,6 +527,49 @@ object HostLatencyHistogram {
   }
 }
 
+class AddressRangeCounterRequest(implicit p: Parameters) extends NastiBundle {
+  val addr = UInt(nastiXAddrBits.W)
+  val len  = UInt(nastiXLenBits.W)
+  val size = UInt(nastiXSizeBits.W)
+}
+
```

Comment:
Give the member a more descriptive name, or describe what this class does in a comment.

Reply:
They're the same as the names in the NastiReadAddressChannel and NastiWriteAddressChannel bundles. I'll add a comment.